sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -214,13 +217,18 @@ def plot(
                     "to automatically set prevalence_pos_label"
```

Comment:
Above, we have the `line_kwargs`, let's validate it as well even though currently there is no issue with the aliases.